### PR TITLE
Implement add content page

### PR DESCRIPTION
**Pull request by ol0lll**
Implement #152 
Todo:
~~* 'node/add' links~~
~~* path override in admin_ui_support~~

### CODE REVIEW

File: src/components/05_pages/AddContent/AddContent.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    requestContentTypes: PropTypes.func.isRequired,
+  };
+  componentDidMount() {
+    this.props.requestContentTypes();
+  }
+  render = () => {
+    return (
+      <div className={styles.root}>
+        <Paper>
+          {console.log(this.props.contentTypes)}
```
Comment (dawehner) [inline src/components/05_pages/AddContent/AddContent.js]: I guess we could drop that :)

File: src/components/05_pages/AddContent/AddContent.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          <List>
+            {Object.keys(this.props.contentTypes).map(contentType => (
+              <ListItem key={contentType}>
+                <ListItemText
+                  primary={this.props.contentTypes[contentType].name}
+                  secondary={(<div dangerouslySetInnerHTML={{ __html: this.props.contentTypes[contentType].description}} />)}
```
Comment (dawehner) [inline src/components/05_pages/AddContent/AddContent.js]: 🔧 Note: We do have the ```interweave``` module which strips dangerous HTML from the description

**Comment by ol0lll** [comment]
I moved the page route to _content/add_ because I could not figure out how to make subpaths like _node/add/<content_type>_ work when the app is overriding _node/add_. Do we need an issue for that?

**Comment by dawehner** [comment]
How about applying this PR on top of it: https://github.com/ol0lll/drupal-admin-ui/pull/1

File: admin_ui_support/admin_ui_support.routing.yml [hunk 3]
```diff
@@ -61,3 +61,13 @@ admin_ui_support.system.admin_content:
     _admin_related_route: 'system.admin_content'
   requirements:
     _permission: 'access administration pages'
+
+admin_ui_support.node.add_page:
```
Comment (dawehner) [inline admin_ui_support/admin_ui_support.routing.yml]: Thank you for adding it!

**Pull request merged**